src/main/kotlin/com/github/sidedev/sidekick/api/SidekickService.kt

```diff
@@ -28,6 +39,11 @@ class SidekickService(
                 },
             )
         }
+        install(WebSockets) {
+            // NOTE: DO NOT REPLACE THIS WITH pingIntervalMillis THAT DOES NOT WORK!
+            // That will break tests, so just leave this line alone, keep it as-is and DO NOT REMOVE THIS COMMENT
```

Comment:
lol, the AI kept reverting this so I was desperate. We can remove this comment now.